src/test/kotlin/org/rust/ide/core/overrideImplement/OverrideTraitMembersTest.kt

```diff
@@ -0,0 +1,82 @@
+package org.rust.ide.core.overrideImplement
+
+import junit.framework.TestCase
+import org.intellij.lang.annotations.Language
+import org.rust.lang.RsTestBase
+import org.rust.lang.core.psi.RsImplItem
+import org.rust.lang.core.psi.util.parentOfType
+import java.util.*
+
+class OverrideTraitMembersTest : RsTestBase() {
+    override val dataPath = ""
+
+    fun test1() = doTest("""
+        trait T {
+            fn foo();
+            fn bar() {}
+        }
+        struct S;
+        impl T for S {
+            /*caret*/
+        }
+    """, """
+        *s foo()
+           bar()
+    """, """
+        
+    """)
+
+    private fun doTest(@Language("Rust") code: String,
+                       chooser: String,
+                       @Language("Rust") expected: String) {
+        checkByText(code, expected) {
+            val impl = myFixture.elementAtCaret.parentOfType<RsImplItem>()
+                    ?: fail("Caret is not in an impl block")
```

Comment:
This approach has a problem: JUnit's `fail` returns `Unit` instead of `Nothing`, so I had to write a helper method for that.